storage/PackedRowStoreTupleStorageSubBlock.cpp

```diff
@@ -193,6 +181,7 @@ tuple_id PackedRowStoreTupleStorageSubBlock::bulkInsertTuples(ValueAccessor *acc
       const bool fast_copy =
           (accessor->getImplementationType() ==
               ValueAccessor::Implementation::kCompressedPackedRowStore);
+      const std::size_t attrs_total_size = relation_.getMaximumByteLength();
```

Comment:
Check out the method `addAttribute` in CatalogRelationSchema. Internally it updates this value with
```c++
max_byte_length_ += attr_type.maximumByteLength();
```